Tasks/PublishTestResultsV2/task.json

```diff
@@ -1,158 +1,158 @@
 {
-    "id": "0B0F01ED-7DDE-43FF-9CBB-E48954DAF9B1",
-    "name": "PublishTestResults",
-    "friendlyName": "Publish Test Results",
-    "description": "Publish test results to Azure Pipelines",
-    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/test/publish-test-results",
-    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=613742)",
-    "category": "Test",
-    "visibility": [
-        "Build",
-        "Release"
-    ],
-    "author": "Microsoft Corporation",
-    "version": {
-        "Major": 2,
-        "Minor": 247,
-        "Patch": 0
+  "id": "0B0F01ED-7DDE-43FF-9CBB-E48954DAF9B1",
```

Comment:
The only change in this file should be on the line 16(something similar to task.loc.json). Can you verify once through file diff, that we are not missing anything while updating this file?

Reply:
verified. No changes at rest places